### ISSUE TITLE
Admin Menu is Almost Unusable

### ISSUE THREAD

**Issue by Mixddux** [rjcalifornia/architect-theme#1]
So close! The theme is looking good but lots of UI issues.

Are you still working on this?

**Comment by rjcalifornia** [comment]
Yes, I’m working on the theme, fixing some issues with the profile page. Could you provide a screenshot of the problem?

**Comment by vcnwafor** [comment]
I think removing most of the calls to deprecated functions elgg_get_menu_item in (3.3.8) will solve the ui issues.

**Comment by rjcalifornia** [comment]
I see, I'm still in the process of upgrading my plugins and themes to 3.3.8 so I think by November I will release a newer version of this theme.
